service/diagnosticv2/types.go

```diff
@@ -184,105 +228,19 @@ type CurlRequest struct {
 	UserAgent string `json:"userAgent"`
 }
 
-/*
-type DTGTMPropertiesResp struct {
+type GTMPropertiesResp struct {
 	GtmProperties []struct {
 		Property string `json:"property"`
 		Domain   string `json:"domain"`
 		HostName string `json:"hostName"`
 	} `json:"gtmProperties"`
 }
 
-type DTGTMPropertyIpsResp struct {
+type GTMPropertyIpsResp struct {
```

Comment:
Try to keep naming convention bit more clear - look at what has been created already as structs in this file.
Lets avoid cryptic `ABCRespCliXyz` types